src/components/Question.jsx

```diff
@@ -207,7 +207,7 @@ export default class Question extends React.Component {
 
     if (this.context.options.debugMode && !FormUtils.hasAnswer(question)) {
       return (
-          <Accordion activeKey={this.state.expanded ? question['@id'] : undefined} className="answerable-section">
+          <Accordion activeKey={this.state.expanded ? question['@id'] : this.setState({expanded: true})} className="answerable-section">
```

Comment:
hvery strange, what does this code actually mean ? Very strange place to run setState(). Will be called on each render() method call ...

So why not only having <Accordion activeKey={question['@id']} -- how is solution that you proposed better ?

Reply:
Yes indeed, I just did not think about it